test/dymaptic.GeoBlazor.Core.Test.Blazor.Shared/Components/AuthenticationManagerTests.cs

```diff
@@ -0,0 +1,148 @@
+using dymaptic.GeoBlazor.Core.Model;
+using Microsoft.AspNetCore.Components;
+using Microsoft.Extensions.Configuration;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+
+namespace dymaptic.GeoBlazor.Core.Test.Blazor.Shared.Components;
+
+public class AuthenticationManagerTests: TestRunnerBase
```

Comment:
Can you add some comments to this class about setting up the secrets/appsettings?